python/wikibot/wikibot.py

```diff
@@ -197,8 +255,9 @@ def ever_had_prop(self, wd_item, prop):
     return False
 
   def ever_had(self, wd_item, prop, target):
-    # Up to 150 revisions covers the full history of 99% human items
-    revisions = wd_item.revisions(total=150, content=True)
+    in_deletions = self.deletions.contains(wd_item.title(), prop, target)
+    if in_deletions: return True
+    revisions = wd_item.revisions(starttime=self.deletions.timestamp(), content=True)
```

Comment:
In this line I get the following error:
 pywikibot.data.api.APIError: badtimestamp_rvstart: Invalid value "<bound method Deletions.timestamp of <__main__.Deletions object at 0x7f09f7deefd0>>" for timestamp parameter "rvstart". [help:See https://www.wikidata.org/w/api.php for API usage. Subscribe to the mediawiki-api-announce mailing list at &lt;https://lists.wikimedia.org/mailman/listinfo/mediawiki-api-announce&gt; for notice of API deprecations and breaking changes.]
CRITICAL: Exiting due to uncaught exception <class 'pywikibot.data.api.APIError'>